templates/admin/header.html

```diff
@@ -0,0 +1,26 @@
+
+<div class='admin-header row'>
+  <div class='col col--grow'>
+    <div class='admin-header__name'>
+      <p>ADMIN CONSOLE</p>
```

Comment:
Now that we're keeping this and have style applied, thoughts on making the case normal 
```suggestion
      <p>Admin Console</p>
```

And in the CSS applying:
```
text-transform: uppercase
```
https://caniuse.com/mdn-css_properties_text-transform

The upper case is more of a style/presentation thing than actual meaning and feels more appropriate to have in the CSS.